src/collectd.conf.in

```diff
@@ -582,6 +582,7 @@
 #<Plugin interface>
 #	Interface "eth0"
 #	IgnoreSelected false
+#	ActiveInterfaceOnly false
```

Comment:
I'm a bit unhappy about the option name. How do you feel about *ReportInactive*?